<?xml version="1.0"?>

<launch>
    <arg name="rviz_config" default="cam_traj_opt"/>
    <arg name="rviz" default="true"/>

    <param name="use_sim_time" value="false"/>

<!--     <node name="camera_traj_optimization" pkg="trajectory_optimization" type="dwa_perception_aware.py" output="screen"/> -->
    <node name="camera_traj_optimization" pkg="trajectory_optimization" type="camera_traj_optimization.py" output="screen"/>
<!--     <node name="camera_traj_optimization" pkg="trajectory_optimization" type="camera_waypoints_optimization.py" output="screen"/> -->

    <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz" args="-d $(find trajectory_optimization)/config/$(arg rviz_config).rviz"/>
</launch>
